drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
-	callback    driverapi.DriverCallback
```

Comment:
> a design to include the remote drivers statically in libnetwork project.

E.g., `"weave"` hard-wired in here somewhere? I would find that just as unsatisfactory!

The requirement to keep drivers out of the public API constrains us to reaching out to external systems; possibly there could be an indirection via an interface so that it is not _too_ prescriptive, but then synchronisation is potentially a problem.

As you suggest, a PR (or an update to one) for that bit will follow.